subprojects/emcee/android-device/src/main/kotlin/com/avito/android/device/manager/internal/FindAndroidDevice.kt

```diff
@@ -30,7 +31,8 @@ internal class FindAndroidDevice(
                 sdk = sdk,
                 type = type,
                 serial = DeviceSerial(foundDevice.serial),
-                adb = adb
+                adb = adb,
+                installPackage = InstallPackage(adb),
```

Comment:
May be `InstallPackageRequest.install(app, serial)` ? Or `InstallPackageCommand`